libraries/AP_InertialSensor/AP_InertialSensor_Invensensev3.cpp

```diff
@@ -578,6 +578,9 @@ void AP_InertialSensor_Invensensev3::set_filter_and_scaling(void)
     register_write(INV3REG_ACCEL_CONFIG0, odr_config);
 
     // setup anti-alias filters for gyro at 1/4 ODR, notch left at default
+    // The defaults for 42605 and 42609 are different to the 42688, make sure AAF and notch are enabled on all
+    uint8_t aaf_enable = register_read_bank(1, INV3REG_GYRO_CONFIG_STATIC2);
+    register_write_bank(1, INV3REG_GYRO_CONFIG_STATIC2, aaf_enable | 0x03);
```

Comment:
This is doing the opposite since the lower 2 bits mean disabled, should be something like this: `aaf_enable & ~0x03`.

Reply:
It's late, you are right